src/components/App/App.js

```diff
@@ -22,7 +22,7 @@ const App = () => {
           <Route path='/profiles' component={Profiles} />
           <Route path='/passport' component={Passport} />
           {/* below is test-only for specific incident view */}
-          <Route path='/incident-details/:id' render={({ match }) => {
+          <Route path='/incident/details/:id' render={({ match }) => {
```

Comment:
Interesting, I have been using the other path.

Reply:
Same. I like this new path naming convention better, however.